automation_py/automations/gmail_creator/account_setup.py

```
@@ -1035,98 +937,53 @@ def _select_gender_neutral(self):
+                "//div[contains(text(), 'Create a password')]",
+                "//div[contains(text(), 'Criar uma senha')]"
             ]
-
             for element in password_elements:
                 try:
-                    if WebDriverWait(self.driver, 2).until(EC.presence_of_element_located((By.XPATH, element))):
-                        logger.info("[OK] Detectada tela de senha")
+                    if WebDriverWait(self.driver, 1).until(EC.presence_of_element_located((By.XPATH, element))):
                         return "password_screen"
-                except TimeoutException:
+                except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)